desc/protoparse/ast/service.go

```diff
@@ -0,0 +1,273 @@
+package ast
+
+import "fmt"
+
+// ServiceNode represents a service declaration. Example:
+//
+//  service Foo {
+//    rpc Bar (Baz) returns (Bob);
+//    rpc Frobnitz (stream Parts) returns (Gyzmeaux);
+//  }
+type ServiceNode struct {
+	compositeNode
+	Keyword    *KeywordNode
+	Name       *IdentNode
+	OpenBrace  *RuneNode
+	Decls      []ServiceElement
+	CloseBrace *RuneNode
+}
+
+func (*ServiceNode) fileElement() {}
+
+// NewServiceNode creates a new *EnumNode. All arguments must be non-nil.
```

Comment:
```suggestion
// NewServiceNode creates a new *ServiceNode. All arguments must be non-nil.
```

Reply:
Good catch! Thanks for the thorough review of doc comments!